docs/dictionary.txt

```diff
@@ -0,0 +1,78 @@
+Async
+CPython
+Cobertura
+Cython
+Dataclasses
+Docstring
+Enums
+Indices
+JUnit
+Metaclass
+Metaclasses
+Posix
+Pythonic
+Typeshed
+Unfollowed
+Zulip
+api
+arity
+async
+attr
+attrs
+builtin
+bytecode
+cdef
+codebase
+config
+contravariant
+contravariantly
+covariantly
+cpdef
+deserialization
+devs
+dict
+dmypy
+downloadable
+fallback
+genericity
+indices
+ing
```

Comment:
This is for ”await-ing” in https://github.com/python/mypy/blob/f5058bcb269a55b33a02c3e3fd345541839bf06e/docs/source/more_types.rst#L576